skalko.oksana/2homework/brackets.rb

```diff
@@ -0,0 +1,30 @@
+class String
+  def valid?
+    brack = self.chars
+    hash = {"{" => "}", "[" => "]", "(" => ")", "<" => ">"}
```

Comment:
Add space after the opening parenthesis and before closing parenthesis
```suggestion
    hash = { '{' => '}', '[' => ']', '(' => ')', '<' => '>' }
```